src/wallet/wallet.h

```diff
@@ -1157,7 +1157,7 @@ class CWallet final : public FillableSigningProvider, private interfaces::Chain:
      * @param orderForm[in] BIP 70 / BIP 21 order form details to be set on the transaction.
      * @param state[in,out] CValidationState object returning information about whether the transaction was accepted
      */
-    bool CommitTransaction(CTransactionRef tx, mapValue_t mapValue, std::vector<std::pair<std::string, std::string>> orderForm, CValidationState& state);
+    void CommitTransaction(CTransactionRef tx, mapValue_t mapValue, std::vector<std::pair<std::string, std::string>> orderForm, CValidationState& state);
```

Comment:
in commit 1f490c9a5f8767477f6ffe382b4a0578dbac5e3e you claim that the return value is removed, but you still return `state`, which is used in a single place (`feebumper`). Haven't thought too much about this, but it seems fine to use there because it can't lead to loss of funds?

Also, that code path shouldn't be hit unless there is a sudden massive fee spike or a logic error.

Reply:
I've removed `state`, since it's never actually set.

Reply:
ouch. Good catch :grimacing: